pkg/xds/experimental/client/cell.go

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+package client
+
+import (
+	"context"
+	"log/slog"
+
+	"github.com/cilium/hive/cell"
+	"github.com/cilium/hive/job"
+	"github.com/spf13/pflag"
+	"google.golang.org/grpc"
+)
+
+var Cell = cell.Module(
+	"xds-client",
+	"xDS client",
+
+	//exhaustruct:ignore
+	cell.Config(Config{Options: *Defaults}),
+	cell.Provide(NewXDSClient),
+)
+
+type Config struct {
+	Addr string
```

Comment:
These fields don’t match with the flag names. You could either rename the fields (e.g. call this ”XDSServerAddress”), or use mapstructure struct tag. https://docs.cilium.io/en/latest/contributing/development/hive/#api-config

Reply:
Changed to `ServerAddr`. I think package path implies xds.

Currently flag definition and usage is very close together (in the same file), so I believe that long names are no longer necessary

Reply:
Done. Thanks @ovidiutirla for explaining it to me